src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String SPECIAL_CHAR = ",";
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+
+    private StringBuilder getFileStat(String fromFileName) {
+        File fileToRead = new File(fromFileName);
+        String[] text;
+        String value;
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileToRead))) {
+            int supply = 0;
+            int buy = 0;
+            int result = 0;
+            while ((value = bufferedReader.readLine()) != null) {
+                text = value.split(SPECIAL_CHAR);
+                for (String s : text) {
+                    if (s.equals(SUPPLY)) {
+                        supply += Integer.valueOf(text[1]);
+                    } else if (s.equals(BUY)) {
+                        buy += Integer.valueOf(text[1]);
+                    }
+                }
+            }
+            result = supply - buy;
+            stringBuilder.append("supply," + supply + System.lineSeparator())
+                    .append("buy," + buy + System.lineSeparator())
+                    .append("result," + result + System.lineSeparator());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read this file", e);
+        }
+        return stringBuilder;
+    }
 
+    private void getWrittenFile(StringBuilder stringBuilder, String toFileName) {
```

Comment:
pretty much the same with this method - it's too complex and large, try to simplify it, remove everything unnecessary

also the method name is confusing - `void` return type and `get` in method name :)